modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTypeSettings.java

```diff
@@ -164,6 +165,16 @@ public interface NumericDDMFormFieldTypeSettings
 	)
 	public boolean hideField();
 
+	@DDMFormField(
+		dataType = "string", label = "%html-autocomplete-attribute",
+		properties = {
+			"invalidCharacters=[^a-z0-9-]|-{2,}", "visualProperty=true",
+			"maxLength=20"
```

Comment:
Can you sort the order of the properties, please?

Reply:
Done